### PR TITLE
CW persistent bugger

### PR DESCRIPTION
**Pull request by aepaysinger**

### CODE REVIEW

File: code_challenges/code_wars/persistent_bugger.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class PersistentMultiplication:
+    def __init__(self, numbers):
+        self.numbers = numbers
+        self.count = 0
```
Comment (W-Ely) [inline code_challenges/code_wars/persistent_bugger.py:4]:
```suggestion
        self.number = number
        self.count = 0
        self.numbers = []
```

File: code_challenges/code_wars/persistent_bugger.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    def break_up_numbers(self):
+        self.numbers = [self.numbers]
+        self.numbers = [
+            int(x) if x.isdigit() else x for z in self.numbers for x in str(z)
+        ]
```
Comment (W-Ely) [inline code_challenges/code_wars/persistent_bugger.py:10]:
```suggestion
        self.numbers = [
            int(x) for x in str(self.number)
        ]
```

File: code_challenges/code_wars/persistent_bugger.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            return 0
+
+        total = [self.numbers[0]]
+        for i in range(1, len(self.numbers)):
+            total[0] *= self.numbers[i]
+        self.count += 1
+        total = [int(x) if x.isdigit() else x for z in total for x in str(z)]
+        self.numbers = total
+        if len(self.numbers) > 1:
+            self.multiply_numbers()
```
Comment (W-Ely) [inline code_challenges/code_wars/persistent_bugger.py:25]:
```suggestion
        total = self.numbers[0]
        for i in range(1, len(self.numbers)):
            total *= self.numbers[i]
        self.count += 1
        self.numbers = [int(x) for x in str(total)]
        if len(self.numbers) > 1:
            self.multiply_numbers()
```

**Comment by aepaysinger** [comment]
i made some of the changes we discussed. i also changed what total was just not in the way we discussed. let me know your thoughts

**Review by W-Ely — DISMISSED** [review]

**Review by W-Ely — DISMISSED** [review]

**Review by W-Ely — APPROVED** [review]

**Pull request merged**